### PR TITLE
Remove deprecated webfont utilities and update cf-typography and cf-layout

### PR DESCRIPTION
**Pull request by anselmbradford**
## Removals

- Removes deprecated `licensed-fonts.css` and `u-webfont-` utilities.

## Changes

- Updates to `cf-layout` and `cf-typography` 5.x.

## Testing

- Run `npm install && gulp build && gulp watch` and check the demo.
- Demo should be unchanged from master branch (see screenshots below–**TIP: look at the side-by-side images and cross your eyes so they line up and you'll see what differs between them**), other than the "Select time range" is bolder (which I'm unclear why, but should not matter).

## Screenshots

|  before  |  after  |
|---|---|
| download-before | download-after|

### CODE REVIEW

File: src/css/cfpb-chart-builder.less [hunk 1]
```diff
@@ -57,7 +57,7 @@
 
   .chart-label {
     font-size: @base-font-size-px;
-    .u-webfont-regular();
+    font-family: @webfont-regular;
```
Comment (Scotchester) [inline src/css/cfpb-chart-builder.less]: This should be `font-family: inherit;` to revert to the setting of "Avenir Next" on `body`.

Comment (anselmbradford) [reply src/css/cfpb-chart-builder.less]: Fixed! Thanks! How'd you figure that out?

File: gulp/tasks/styles.js [hunk 2]
```diff
@@ -20,14 +21,16 @@ function stylesDemo() {
...
-        'android 4',
-        'BlackBerry 7',
-        'BlackBerry 10' ]
-    } ) )
+    .pipe( gulpPostcss( [
+      autoprefixer( {
+        browsers: [
+          'last 2 version',
+          'not ie <= 8',
+          'android 4',
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply gulp/tasks/styles.js]: This is our original browser support list when we added browser support lists, so the list here is a legacy from that time and should probably be aligned with that in cfgov-refresh.

**Comment by anselmbradford** [comment]
> Additionally, the bolder "Select time range" labels are happening because of this on L285–294:

@Scotchester Looks like this won't work at small sizes unfortunately. I'll leave as is for now: 

screen shot 2018-09-06 at 3 52 23 pm

File: package.json [hunk 3]
```diff
@@ -35,13 +35,14 @@
   "homepage": "https://github.com/cfpb/cfpb-chart-builder#readme",
   "dependencies": {
     "cf-core": "^4.10.0",
-    "cf-layout": "^4.5.1",
-    "cf-typography": "^4.2.1",
+    "cf-layout": "5.0.0",
```
[the remark this answers is not in the document]
Comment (marteki) [reply package.json]: As long as it doesn't break things to remove the carets. I have a vague memory of developers going through multiple repos to add carets to dependencies like this so future major version changes wouldn't mistakenly break live pages, but can't remember specifics. @Scotchester probably would know more.

[the remark this answers is not in the document]
Comment (Scotchester) [reply package.json]: It definitely doesn't break anything to remove them. I don't recall an effort to go through and adding them, @marteki. If anything, we probably went through removing carets, which would be safer than having them.

Third-party dependencies should always be fully pinned. As for our internal dependencies, I go back and forth on whether I think we can trust ourselves to not include a breaking change in a minor release. I'm more confident we can trust ourselves with patch releases, so I'd suggest using the tilde range, instead, which only allows for automatic patch updates.

File: src/css/cfpb-chart-builder.less (hunk 1 above)
Comment (Scotchester) [reply src/css/cfpb-chart-builder.less]: Inspected and saw that your original rule was resulting in `Arial, Arial, sans-serif;`, so the `@webfont-regular` var wasn't set correctly. Removing the rule entirely resulted in things using Highchart's default of Lucida, so the rule has to be there. I knew that cf-core was setting Avenir Next on `body`, so `inherit` restores it to that from the Highcharts style.

**Comment by Scotchester** [comment]
> Looks like this won't work at small sizes unfortunately. I'll leave as is for now

Did you refresh the page after resizing the window? Something about it doesn't adjust properly if you just resize the window.

**Comment by anselmbradford** [comment]
> Did you refresh the page after resizing the window? Something about it doesn't adjust properly if you just resize the window.

Hmmmmm okay, I'll check.

**Comment by anselmbradford** [comment]
Same issue.

**Comment by marteki** [comment]
@anselmbradford What breakpoint did you check the small size at? The existing placement of... well, almost everything in the charts is very exact and near pixel-perfect for a set range of browser widths. It is not truly responsive.

IOW Based on the current state of the code, I expect to see that kind of overlap at certain small screen sizes. 😫

**Comment by Scotchester** [comment]
Hmm... I see why I didn't see the problem before, I wasn't looking at one on a line chart that has the legend.

Very strange. I don't see why my proposed changes would result in that.

**Comment by Scotchester** [comment]
Ah, figured it out. I mishandled the `transform: translateY` rules. This works:

```less
  .highcharts-range-selector-label {
    display: table;
    margin-top: 0 !important;
    width: 100%;
    color: @gray;

    .chart-label();
    transform: translateY( -5px );

    .respond-to-max( 800px, {
        .h6();
        transform: translateY( 415px );
    } );
```

**Comment by anselmbradford** [comment]
PR for Scott's fix at https://github.com/cfpb/cfpb-chart-builder/pull/137

**Pull request merged**